docs/docusaurus/docs/changelog.md

```diff
@@ -10,6 +10,58 @@ title: Changelog
+* [DOCS] Remove jq dependency for building docs ([#7893](https://github.com/great-expectations/great_expectations/pull/7893))
+* [DOCS] Less verbose logging during docs build ([#7894](https://github.com/great-expectations/great_expectations/pull/7894))
+* [DOCS] Corrections to Document "How to request data from a Data Asset" with proper FDS implementation and integration test. ([#7896](https://github.com/great-expectations/great_expectations/pull/7896))
+* [DOCS] Remove or Modify References to the CLI ([#7875](https://github.com/great-expectations/great_expectations/pull/7875))
+* [DOCS] Updated contribution docs ([#7880](https://github.com/great-expectations/great_expectations/pull/7880))
+* [DOCS] Glossary Update ([#7900](https://github.com/great-expectations/great_expectations/pull/7900))
+* [DOCS] Removes filesystem and core block config docs ([#7913](https://github.com/great-expectations/great_expectations/pull/7913))
+* [DOCS] FDS Deployment Pattern Redshift ([#7868](https://github.com/great-expectations/great_expectations/pull/7868))
+* [DOCS] Document and simplify local docs build ([#7892](https://github.com/great-expectations/great_expectations/pull/7892))
+* [DOCS] How to Edit Existing `ExpectationSuite` ([#7859](https://github.com/great-expectations/great_expectations/pull/7859))
+* [DOCS] CLI Clean-up ([#7904](https://github.com/great-expectations/great_expectations/pull/7904))
+* [MAINTENANCE] FDS Documentation - Creating ExpectationSuite with Domain Knowledge ([#7852](https://github.com/great-expectations/great_expectations/pull/7852))
+* [MAINTENANCE] Refactor directory data asset ([#7878](https://github.com/great-expectations/great_expectations/pull/7878))
+* [MAINTENANCE] partial `checkpoint` type checking ([#6914](https://github.com/great-expectations/great_expectations/pull/6914))
+* [MAINTENANCE] type-checking `checkpoint/actions.py` ([#7899](https://github.com/great-expectations/great_expectations/pull/7899))
+* [MAINTENANCE] Trigger docs_integration on develop ([#7902](https://github.com/great-expectations/great_expectations/pull/7902))
+* [MAINTENANCE] Finish type-checking `checkpoint` ([#7901](https://github.com/great-expectations/great_expectations/pull/7901))
+* [MAINTENANCE] Update How to set up GX to work with data in Azure Blob Storage ([#7910](https://github.com/great-expectations/great_expectations/pull/7910))
+* [MAINTENANCE] `CloudDataContext` `ExpectationsStore` can only request one Suite at a time ([#7905](https://github.com/great-expectations/great_expectations/pull/7905))
+* [MAINTENANCE] Removing engine-specific tests that assert generic behavior ([#7918](https://github.com/great-expectations/great_expectations/pull/7918))
+* [MAINTENANCE] Use with open file to reduce open files within tests ([#7906](https://github.com/great-expectations/great_expectations/pull/7906)) (thanks @TrangPham)
```

Comment:
Did you also mean to remove this? Is this an internal user?

Reply:
thanks